extensions.md

```diff
@@ -1515,6 +1515,21 @@ To alleviate this, consider implementing ```-webkit-overflow-scrolling: touch```
 ### Invoking an Extension with parameters (iOS)
 iOS support invocation URLS. An item can be linked to using these URLs.
 
+Example: [onmediafly://9cf282320e6340ee8b830e5376d54531product265958?param1=1&param2=2](onmediafly://9cf282320e6340ee8b830e5376d54531product265958?param1=1&param2=)
```

Comment:
Maybe it's just GitHub's interface, but it looks like you put this new documentation in the middle of the "Invoking an Extension with parameters (iOS)", instead of at the end as a new section?

Reply:
Good catch. Fixed this.